src/language/__tests__/schema-parser-test.js

```diff
@@ -480,6 +639,7 @@ describe('Schema Parser', () => {
           kind: 'InterfaceTypeDefinition',
           name: nameNode('Hello', { start: 10, end: 15 }),
           description: undefined,
+          interfaces: [],
```

Comment:
Do we have an integration test (probably not under the `parser`, but in `buildASTSchema`) for an AST having `interfaces` as undefined?

I know we ran into problems in the past when hand-modeling the AST where, unless you explicitly set `directives: []`, you'd end up getting an undefined reference, even though the flow type allowed it.

Reply:
I just added an assertion for this case in dfc2da9d937987a7a23baf22ab203a8dea228087.